src/AuthService.Grpc/Extensions/EmailValidationRule.cs

```diff
@@ -0,0 +1,25 @@
+﻿using FluentValidation;
+
+namespace AuthService.Grpc.Extensions
+{
+    /// <summary>
+    ///  Validation rule for email.
+    /// </summary>
+    public static class EmailValidationRule
+    {
+        /// <summary>
+        /// Musts the be valid email.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="ruleBuilder">The rule builder.</param>
+        /// <returns>IRuleBuilderOptions</returns>
+        public static IRuleBuilderOptions<T, string> MustBeValidEmail<T>(this IRuleBuilder<T, string> ruleBuilder)
+        {
+            var builderOptions = ruleBuilder
+                .NotNull()
+                .NotEmpty();
```

Comment:
Where is email verification? I see only NotNull and NotEmpty. Any string can pass this validation rule.